src/redisearch_api.h

```diff
@@ -90,6 +91,61 @@ struct RSIdxOptions {
   RSLanguage lang;
 };
 
+struct RSIdxField {
+  char *path;
+  char *name;
+  struct {
+    bool text;
+    bool tag;
+    bool numeric;
+    bool geo;
+  } types;
```

Comment:
why not enum?

Reply:
Because a field can have more than one type.

Reply:
Why can't this be a mask (i.e. `FieldType`)?

Reply:
FieldType is OPAC for the LLAPI. Changing to use `RSFLDTYPE_`